src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
+
+  for (const prop in robot) {
+    if (!newObj[robot[prop]]) {
+      newObj[robot[prop]] = prop;
+      continue;
+    }
+
+    return null;
```

Comment:
it`s better to rewrite
```suggestion
    if (newObj[robot[prop]]) {
      return null;
    }
    newObj[robot[prop]] = prop;
    
```